pkgs/development/libraries/rocksdb/default.nix

```
@@ -65,7 +69,7 @@ stdenv.mkDerivation rec {
 
   preInstall = ''
     mkdir -p $tools/bin
-    cp tools/{ldb,sst_dump} $tools/bin/
+    cp tools/{ldb,sst_dump}${stdenv.buildPlatform.extensions.executable} $tools/bin/
```

Sorry for the regression! This should be hostPlatform.

```suggestion
    cp tools/{ldb,sst_dump}${stdenv.hostPlatform.extensions.executable} $tools/bin/
```